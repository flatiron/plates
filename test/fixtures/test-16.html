<b>Flatiron Components</b>
<br/>
<ul>
  <li class='components'></li>
</ul>
