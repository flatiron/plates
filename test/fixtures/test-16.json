{
  "components": [
    "Union",
    "Director",
    "Broadway",
    "Plates",
    "Resourceful"
  ]
}
