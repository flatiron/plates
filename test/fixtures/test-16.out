<b>Flatiron Components</b>
<br/>
<ul>
  <li class='components'>Union</li><li class='components'>Director</li><li class='components'>Broadway</li><li class='components'>Plates</li><li class='components'>Resourceful</li>
</ul>
